README.md

```diff
@@ -2,6 +2,35 @@
 
 > Our guidelines for how we do Scrum
 
+## Development
+
+Development of an implementation of a backlog item is carried out in accordance to [GitHub Flow](https://guides.github.com/introduction/flow/) with some additional requirements and tooling support. One of the primary requirements of the development workflow is the use of [protected branches](https://help.github.com/articles/configuring-protected-branches/) in order to enforce the workflow and ensure the integrity of the main product branches.
+
+### Workflow
+
+1.  __Create a topic branch__  
+    All work is to be carried out on topic branches to ensure isolation. Make sure that the name of your branch describes the work you'll be doing and that it's written in kebab-case (eg. `my-topic-branch`).
+
+2.  __Add commits__  
+    Once your topic branch has been created you can start making the desired changes. Make sure to keep each commit small in order to allow for easy reverts which _can_ and _will_ happen. It's a good idea to use a [GUI client](https://git-scm.com/download/gui/linux) when committing to ease [staging patches](https://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging#Staging-Patches). Also make sure to familiarize yourself with [the seven rules of a great git commit message](http://chris.beams.io/posts/git-commit/).
+
+3.  __Create a pull request__  
+    When you've committed and pushed some work it's time to create a pull request against the main product branch. Follow the same guidelines for writing pull request titles and descriptions as you would for commit messages. If your pull request resolves an existing issue then make sure to state this in either the title or description of the pull request using the [corresponding keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) to associate the pull request with the issue. Once created, your pull request will show up in "In Progress" stage on the task board. Keep in mind that you can continue committing to your branch after the pull request has been created.
+
```

Comment:
Assignee definitely, not sure about labels yet. It might be a good idea to label issues according to their artifact type (`story`, `task`, `epic`), but I'm not sure what would make the most sense for pull requests.